src/components/ChattingScreen.js

```diff
@@ -1,31 +1,136 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
 import Header from './Header';
 import MessageSender from './MessageSender';
 
 export default function ChattingScreen() {
-	const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
-	const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
-
-	const MSGLIST = [
-		{ user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
-		{ user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
-		{ user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
-		{ user: false, content: '안녕 은아' },
-		{ user: false, content: '뭐해 ?' },
-		{ user: true, content: '시원아 넌 최고의 팀장이야' },
-		{ user: false, content: '나도 아니까  ' },
-		{ user: false, content: '그만 말해줘도 돼' },
-		{ user: true, content: 'ㅠㅠ' },
-	];
-
-	return (
-		<Wrapper>
-			<Header></Header>
-			<MessageSender></MessageSender>
-		</Wrapper>
-	);
+  const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
+  const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
+
+  const MSGLIST = [
+    { user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
+    { user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
+    { user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
+    { user: false, content: '안녕 은아' },
+    { user: false, content: 'ㅁㅎ?' },
+    { user: true, content: '시원아 넌 최고의 팀장이야' },
+    { user: false, content: '나도 아니까  ' },
+    { user: false, content: '그만 말해줘도 돼' },
+    { user: true, content: 'ㅠㅠ' },
+  ];
+
+  const [currentUser, setCurrentUser] = useState(false); // 현재 채팅하는 사람
+  const [newMSGList, setNewMSGList] = useState(MSGLIST); // Re-Rendering하기 위하여 생성
+
+  useEffect(() => {
+    window.scrollBy(0, window.innerHeight);
+  });
```

Comment:
1.  `useEffect`의 두번째 인자로 dependency배열을 넘겨주지않으면, 컴포넌트가 리렌더링될 때마다 호출이 된답니당. 그래서 재용님코드에서 프로필사진을 클릭해서 user를 바꿔보면,` currentUser`의 상태가 바뀌니까 컴포넌트가 리렌더링되므로 그때도 스크롤이 맨 밑으로 내려가게 되어있어요! 
이를 해결하려면 두번째 인자로 dependency배열을 넘겨줘서 그 인자가 변경되었을때만 호출되도록 적용시키면 됩니당

2.  `window.innerHeight` 보다는 `document.body.scrollHeight`을 이용하시는게 좋을 것 같아요! 메시지를 길게 추가하다 보면 `window.innerHeight`적용시 끝까지 스크롤되지 않는걸 확인할 수 있으실거에요ㅎㅎ

```suggestion
  useEffect(() => {
   window.scrollBy(0, document.body.scrollHeight);
  },[newMSGList]);
```

Reply:
1. 예외처리를 깜박했네요! 감사합니다!

2. 오.. 정보 감사합니다..

Reply:
잘 배우고 갑니당